Sources/StreamChatUI/UIConfig+ColorTheme.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+public extension UIConfig {
+    struct StreamColorTheme: ColorTheme {
+        // MARK: - Text
+
+        /// General textColor, should be something that contrasts great with your `background` Color
+        public var text: UIColor = .streamBlack
+
+        /// Static color which should stay the same in dark and light mode, because it's only used as text on small UI Elements
+        /// such as `ChatUnreadCountView`, `GiphyBadge` or Commands icon.
+        public var staticColorText: UIColor = .streamWhiteStatic
+        public var subtitleText: UIColor = .streamGray
+
+        // MARK: - Text interactions
+
+        public var highlightedColorForColor: (UIColor) -> UIColor = { $0.withAlphaComponent(0.5) }
+        public var disabledColorForColor: (UIColor) -> UIColor = { _ in .lightGray }
+        public var unselectedColorForColor: (UIColor) -> UIColor = { _ in .lightGray }
+
+        // MARK: - Background
+
+        public var background: UIColor = .streamWhiteSnow
+        public var background1: UIColor = .streamWhiteSmoke
+        public var background2: UIColor = .streamGrayGainsboro
+        public var background3: UIColor = .streamOverlay
+        public var background4: UIColor = .streamOverlayDark
+        public var background5: UIColor = .streamWhite
```

Comment:
I actually kinda like this pattern but we should just be sure the naming makes sense, i.e. `background1` is lighter than `background2` etc ...

Reply:
Actually this kinda magically works. i.e. If you look into our designs, it is exactly like you say (1 is lighter than 2 etc.)